src/helpers/matchers/accessibilityState.ts

```diff
@@ -0,0 +1,39 @@
+import { AccessibilityState } from 'react-native';
+import { ReactTestInstance } from 'react-test-renderer';
+
+/**
+ * Default accessibility state values based on experiments using accessibility
+ * inspector/screen reader on iOS and Android.
+ *
+ * @see https://github.com/callstack/react-native-testing-library/wiki/Accessibility:-State
+ */
+const defaultState: AccessibilityState = {
+  disabled: false,
+  selected: false,
+  checked: undefined,
+  busy: false,
+  expanded: undefined,
+};
+
+export function matchAccessibilityState(
+  node: ReactTestInstance,
+  matcher: AccessibilityState
+) {
+  const state = node.props.accessibilityState;
+  return (
```

Comment:
Maybe we could use accessibilityStateKeys to refactor this part, something like this maybe ?

```ts
accessibilityStateKeys.forEach(accessibilityStateKey => {
   if (!matchState(state, matcher, accessibilityStateKey) {
       return false;
} 

return true;
 }
```

Reply:
`accessibilityStateKeys.every(key => matchState(state, matcher, key))` probably. It should do short circuiting by itself without ifs and early returns.